src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/UpgradeOptions.cs

```diff
@@ -23,6 +23,8 @@ public class UpgradeOptions
 
         public bool Verbose { get; set; }
 
+        public bool ReadinessAcknowledgement { get; set; }
```

Comment:
The naming of this option is a little confusing to me. Something like 'BypassChecks' or 'Force' seems clearer.

Reply:
Renamed